lib/txmgr/metrics.go

```diff
@@ -32,6 +32,6 @@ var (
 		Subsystem: "txmgr",
 		Name:      "tx_gas_used",
 		Help:      "Gas used by transactions",
-		Buckets:   []float64{10000, 25000, 50000, 100000, 250000, 500000, 1000000},
+		Buckets:   []float64{21_000, 50_000, 100_000, 250_000, 500_000, 1_000_000, 5_000_000, 10_000_000},
```

Comment:
If these numbers are not meaningful you might as well use [ExponentialBucketsRange](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#ExponentialBucketsRange): 
```suggestion
		Buckets:   prometheus.ExponentialBucketsRange{21_000, 10_000_000, 10),
```